custom_components/adaptive_lighting/__init__.py

```diff
@@ -20,6 +20,21 @@
 
 PLATFORMS = ["switch"]
 
+# Keep service call 'change_switch_settings' updated directly from strings.json
+# import json
```

Comment:
Let’s remove commented out code. It’sa bad practice to keep code as comments around because they are not tested/executed.